app/routers/auth.py

```
@@ -51,6 +55,90 @@ async def kakao_callback(code: str = Query(...), db: AsyncSession = Depends(get_
+        return {
+            "user": {
+                "id": user.id,
+                "name": user.name,
+                "email": user.email,
+            },
+            "access_token": access_token,
+            "refresh_token": refresh_token,
+        }
+
+    except Exception as e:
+        raise HTTPException(status_code=400, detail=f"Login failed: {str(e)}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)